jdaviz/utils.py

```diff
@@ -351,15 +377,66 @@ def flux_conversion(spec, values, original_units, target_units):
             eqv_in = [min(fac), max(fac)]
         else:
             eqv_in = fac
-
         eqv += _eqv_pixar_sr(np.array(eqv_in))
 
+        # transitive units cannot be directly converted, and require
+        # additional conversions to reach the desired end unit.
+        # if spec_unit in [original_units, target_units]:
+        values, updated_units, unit_type = _transitive_conversion(
+            values=values, orig_units=orig_units, targ_units=targ_units,
+            eqv=eqv, spec_unit=spec_unit
+            )
+        if unit_type and unit_type == 'targ':
+            targ_units = updated_units
+        elif unit_type and unit_type == 'orig':
+            orig_units = updated_units
```

Comment:
There's no case for `unit_type` being `None`, so we don't need that in the if statements right?

Reply:
I originally had a case where it did but forgot to remove that check since then, these 2 checks will be removed